server/src/main/kotlin/com/kamelia/jellyfish/rest/auth/Routes.kt

```diff
@@ -21,6 +24,11 @@ private fun Route.login() = postOrCatch<LoginDTO>(path = "/login") { body ->
     call.respond(AuthService.verify(body.username, body.password))
 }
 
+private fun Route.logoutAll() = postOrCatch(path = "/logout") {
+    val username = jwt.subject
+    call.respond(AuthService.logoutAll(username))
+}
```

Comment:
Since we chose to use `PATCH` on `/login` to refresh a session, we should user `DELETE` on `/login` to "destroy" all sessions. We can change the refresh route as well..